sdks/android/Makefile

```diff
@@ -192,14 +192,14 @@ package: app/assets/asm/monodroid_mini_test.dll
 
 ## Package Mono.Debugger.Soft test assemblies
 
-.PHONY: $(TOP)/mcs/class/Mono.Debugger.Soft/dtest-app.exe $(TOP)/mcs/class/Mono.Debugger.Soft/dtest-app.pdb
-$(TOP)/mcs/class/Mono.Debugger.Soft/dtest-app.exe $(TOP)/mcs/class/Mono.Debugger.Soft/dtest-app.pdb:
-	$(MAKE) PROFILE='monodroid' -C $(TOP)/mcs/class/Mono.Debugger.Soft dtest-app.exe
+.PHONY: $(TOP)/mcs/class/lib/monodroid_tools/tests/dtest-app.exe $(TOP)/mcs/class/lib/monodroid_tools/tests/dtest-app.pdb
```

Comment:
`dtest-app.exe` should come from `monodroid` (since we run `dtest-app.exe` on the device, not the host).